applications/StructuralMechanicsApplication/custom_elements/cable_element_3D2N.cpp

```diff
@@ -120,8 +120,8 @@ void CableElement3D2N::UpdateInternalForces(
     }
 
     Vector temp_internal_stresses = ZeroVector(msLocalSize);
-    ProcessInfo temp_process_information;
-    ConstitutiveLaw::Parameters Values(GetGeometry(),GetProperties(),temp_process_information);
+    // ProcessInfo temp_process_information; this mus be passed to the function!

```

Comment:
```suggestion
    // ProcessInfo temp_process_information; this must be passed to the function!

```

Reply:
this is a WIP comment for @KlausBSautter, it should be removed before merging